### PR TITLE
Update Copy/Paste details - group widget functionality

### PR DESCRIPTION
**Pull request by MirandaKapin**
### What does this PR do?
Adds more information around the difference in functionality between copy/pasting for Timeboard and Screenboard groups

https://docs-staging.datadoghq.com/MirandaKapin/group-widget-details/graphing/widgets/#copy-paste-widgets

### CODE REVIEW

File: content/en/graphing/widgets/_index.md [hunk 1]
```diff
@@ -51,16 +51,24 @@ In full screen mode, you can:
 
 Additional options are available for [timeseries widgets][1].
 
-## Copy / paste widgets
+## Copy / Paste Widgets
```
Comment (ruthnaebeck) [inline content/en/graphing/widgets/_index.md]:
```suggestion
## Copy / paste widgets
```
According to our guidelines, h2 headings are sentence case

File: content/en/graphing/widgets/_index.md [hunk 2]
```diff
@@ -51,16 +51,24 @@ In full screen mode, you can:
...
 
 The copied widgets can be pasted within Datadog by using `Ctrl + V` (`Cmd + V` for Mac) on:
 
 * **Dashboards**: Adds a new widget positioned under your mouse cursor.
 * **Notebooks**: Adds a new cell at the end of the notebook.
 
-A copied widget can also be shared outside of Datadog as a static image stored in the same Datadog datacenter as your Datadog site (US or EU).
+When pasting into your favorite chat program that displays link previews (like Slack or Microsoft Teams) a snapshot image of your graph will be displayed along with a direct link to your widget.
+
+### Groups of Widgets
```
Comment (ruthnaebeck) [inline content/en/graphing/widgets/_index.md]:
```suggestion
### Groups of widgets
```

File: content/en/graphing/widgets/_index.md [hunk 3]
```diff
@@ -51,16 +51,24 @@ In full screen mode, you can:
...
-Widgets can be copied on [Dashboards][2], [Notebooks][3], [APM Service][4], and the [APM resource][5] page by using `Ctrl + C` (`Cmd + C` for Mac). Similarly, groups of widgets can be copied on timeboards by selecting the group widget OR on screenboards by `shift + click` on multiple graphs.
+Widgets can be copied on [Dashboards][2], [Notebooks][3], [APM Service][4], and the [APM resource][5] page by using `Ctrl + C` (`Cmd + C` for Mac), or by selecting the share icon and choosing "Copy". 
 
 The copied widgets can be pasted within Datadog by using `Ctrl + V` (`Cmd + V` for Mac) on:
 
 * **Dashboards**: Adds a new widget positioned under your mouse cursor.
 * **Notebooks**: Adds a new cell at the end of the notebook.
 
-A copied widget can also be shared outside of Datadog as a static image stored in the same Datadog datacenter as your Datadog site (US or EU).
+When pasting into your favorite chat program that displays link previews (like Slack or Microsoft Teams) a snapshot image of your graph will be displayed along with a direct link to your widget.
```
Comment (ruthnaebeck) [inline content/en/graphing/widgets/_index.md]:
```suggestion
You can also paste the widget into your favorite chat program that displays link previews (like Slack or Microsoft Teams). This displays a snapshot image of your graph along with a direct link to your widget.
```

File: content/en/graphing/widgets/_index.md [hunk 4]
```diff
@@ -51,16 +51,24 @@ In full screen mode, you can:
...
+
+### Groups of Widgets
+
+Timeboard Group widgets can be copied by hovering over the group widget area and using `Ctrl + C` (`Cmd + C` for Mac) or by selecting the share icon and choosing "Copy".
```
Comment (ruthnaebeck) [inline content/en/graphing/widgets/_index.md]:
```suggestion
Timeboard group widgets can be copied by hovering over the group widget area and using `Ctrl + C` (`Cmd + C` for Mac) or by selecting the share icon and choosing "Copy". 
```

File: content/en/graphing/widgets/_index.md [hunk 5]
```diff
@@ -51,16 +51,24 @@ In full screen mode, you can:
...
 * **Dashboards**: Adds a new widget positioned under your mouse cursor.
 * **Notebooks**: Adds a new cell at the end of the notebook.
 
-A copied widget can also be shared outside of Datadog as a static image stored in the same Datadog datacenter as your Datadog site (US or EU).
+When pasting into your favorite chat program that displays link previews (like Slack or Microsoft Teams) a snapshot image of your graph will be displayed along with a direct link to your widget.
+
+### Groups of Widgets
+
+Timeboard Group widgets can be copied by hovering over the group widget area and using `Ctrl + C` (`Cmd + C` for Mac) or by selecting the share icon and choosing "Copy". 
+*Note: When pasting to Screenboards or Notebooks, individual widgets within the group will be pasted*
```
Comment (ruthnaebeck) [inline content/en/graphing/widgets/_index.md]:
```suggestion
**Note**: When pasting graphs to screenboards or notebooks, individual widgets within the group are pasted.
```

File: content/en/graphing/widgets/_index.md [hunk 6]
```diff
@@ -51,16 +51,24 @@ In full screen mode, you can:
...
+
+To copy multiple screenboard widgets (edit mode only), `shift + click` on multiple widgets and using `Ctrl + C` (`Cmd + C` for Mac).
```
Comment (ruthnaebeck) [inline content/en/graphing/widgets/_index.md]:
```suggestion
To copy multiple screenboard widgets (edit mode only), `shift + click` on the widgets and use `Ctrl + C` (`Cmd + C` for Mac). 
```

File: content/en/graphing/widgets/_index.md [hunk 7]
```diff
@@ -51,16 +51,24 @@ In full screen mode, you can:
...
-A copied widget can also be shared outside of Datadog as a static image stored in the same Datadog datacenter as your Datadog site (US or EU).
+When pasting into your favorite chat program that displays link previews (like Slack or Microsoft Teams) a snapshot image of your graph will be displayed along with a direct link to your widget.
+
+### Groups of Widgets
+
+Timeboard Group widgets can be copied by hovering over the group widget area and using `Ctrl + C` (`Cmd + C` for Mac) or by selecting the share icon and choosing "Copy". 
+*Note: When pasting to Screenboards or Notebooks, individual widgets within the group will be pasted*
+
+To copy multiple screenboard widgets (edit mode only), `shift + click` on multiple widgets and using `Ctrl + C` (`Cmd + C` for Mac). 
+*Note: This only works when sharing within Datadog, this will not generate a preview image outside of Datadog*
```
Comment (ruthnaebeck) [inline content/en/graphing/widgets/_index.md]:
```suggestion
**Note**: This only works when sharing within Datadog. It does generate a preview image outside of Datadog.
```

File: content/en/graphing/widgets/_index.md [hunk 8]
```diff
@@ -51,16 +51,26 @@ In full screen mode, you can:
...
+
+### Groups of Widgets
+
+Timeboard Group widgets can be copied by hovering over the group widget area and using `Ctrl + C` (`Cmd + C` for Mac) or by selecting the share icon and choosing "Copy". 
+
+*Note*: When pasting to Screenboards or Notebooks, individual widgets within the group will be pasted
+
+To copy multiple screenboard widgets (edit mode only), `shift + click` on multiple widgets and using `Ctrl + C` (`Cmd + C` for Mac). 
+
+*Note*: This only works when sharing within Datadog, this will not generate a preview image outside of Datadog
```
Comment (ruthnaebeck) [inline content/en/graphing/widgets/_index.md]:
```suggestion
**Note**: This only works when sharing within Datadog. It does generate a preview image outside of Datadog.
```

File: content/en/graphing/widgets/_index.md [hunk 9]
```diff
@@ -51,16 +51,26 @@ In full screen mode, you can:
...
 * **Notebooks**: Adds a new cell at the end of the notebook.
 
-A copied widget can also be shared outside of Datadog as a static image stored in the same Datadog datacenter as your Datadog site (US or EU).
+When pasting into your favorite chat program that displays link previews (like Slack or Microsoft Teams) a snapshot image of your graph will be displayed along with a direct link to your widget.
+
+### Groups of Widgets
+
+Timeboard Group widgets can be copied by hovering over the group widget area and using `Ctrl + C` (`Cmd + C` for Mac) or by selecting the share icon and choosing "Copy". 
+
+*Note*: When pasting to Screenboards or Notebooks, individual widgets within the group will be pasted
```
Comment (ruthnaebeck) [inline content/en/graphing/widgets/_index.md]:
```suggestion
**Note**: When pasting graphs to screenboards or notebooks, individual widgets within the group are pasted.
```

File: content/en/graphing/widgets/_index.md (hunk 8 above)
Comment (MirandaKapin) [reply content/en/graphing/widgets/_index.md]: @ruthnaebeck this is untrue It should be "It does NOT"

**Pull request merged**